redis/redis.go

```diff
@@ -187,3 +187,27 @@ type SlowLog struct {
 	// ClientName is the name set via the CLIENT SETNAME command (4.0 only).
 	ClientName string
 }
+
+// Latency represents a redis LATENCY LATEST
+type Latency struct {
+	// Latency event name
+	Event string
+
+	// Time is the unix timestamp at which the event was processed.
+	Time time.Time
+
+	// LatestExecutationTime is the amount of time needed for the command execution.
+	LatestExecutionTime time.Duration
+
+	// Records the all-time maximum latency for every element
+	MaxExecutionTime time.Duration
```

Comment:
Similar to above keeping it simple for the user.
```suggestion
	// Latest is the latest recorded latency for the named event.
	Latest time.Duration

	// Max is the maximum latency for the named event.
	Max time.Duration
```